js/popup.js

```diff
@@ -0,0 +1,235 @@
+       <li><span class="divider"></span></li>
+       <li class="tag">html</li>
+       <li><span class="divider"></span></li>
+       <li class="tag">Bootsrap</li>
+       <li><span class="divider"></span></li>
+       <li class="tag">Ruby</li>
+    </ul>
+    <button type="button" class="btn" onclick="">See project</button>
+  </div>
+</div>
+</section>
+<section id="work-grid-four">
+<div class="project-grid">
+    <div class="img-text">
+      <h3 class="project-title">Multi-post Stories</h3>
+       <p class="project-intro">A daily selection of privately personalized reads;
+       no accounts or sign-ups required.
+       has been the industry's standard dummy text ever since the 1500s,
+       when an unknown printer took a standard dummy text.</p>
+     <ul class="tag-list">
+       <li class="tag">css</li>
+       <li><span class="divider"></span></li>
+       <li class="tag">html</li>
+       <li><span class="divider"></span></li>
+       <li class="tag">Bootsrap</li>
+       <li><span class="divider"></span></li>
+       <li class="tag">Ruby</li>
+    </ul>
+     <button type="button" class="btn" onclick="">See project</button>
+   </div>
+   <img class="project-img" alt="project pic" src="https://schemazone.com/wp-content/uploads/2021/03/Project-Manager.jpg">
+ </div>
+</section>
+`;
+});
+
+const modal = document.querySelector('#Work-card-popup');
+for (let i = 0; i < worksArr.length; i += 1) {
+  modal.innerHTML = `<div class="popup">
+    <div class="first-block">
+       <h2 class="work-card-title">${worksArr[i].Title}</h2>
+       <p class="x-icon">${worksArr[i].exitButton}</p>
+     </div>
+     <div class="images-container">
+    <div class="PI"></div>
+    <div class="other-works">
+      <div class="first-project"></div>
+      <div class="first-project"></div>
+      <div class="first-project"></div>
+      <div class="first-project"></div>
+    </div>
+    </div>
+    <span onclick="function next()"class=""><i class="bi bi-chevron-compact-right"></i></span>
+    <span onclick="function prev()"class=""><i class="bi bi-chevron-compact-left"></i></span>
+     <p class="workText">${worksArr[i].firstText}</p>
+     <p class="workText2">${worksArr[i].secondText}</p>
+
+          <ul class="project-tags">
+             <li class="work-tag">${worksArr[i].Skills}</li>
+           </ul>
+       <div class="btns">
+          <button type="button" onclick="" class="btn1">
+            ${worksArr[i].liveDemo}
+            <i class="bi bi-arrow-up-right-circle"></i>
+          </button>
+          <button type="button" onclick="" class="btn2">
+            ${worksArr[i].DemoSource}
+            <img src="images/git-icon.svg" class="icons">
+          </button>
+         </div>
+         <div class="go-to">
+            <span onclick="function prev()"class="previous">${worksArr[i].previousProject}</span>
+            <span onclick="function next()" class="next">${worksArr[i].nextProject}</span>
+         </div>
+         </div>
+`;
+}
```

Comment:
- [ ] Good job so far your popup is displaying :+1: But it is not working as expected :cry:  For both Desktop and mobile versions your popup does not conform to the dimensions set in the Figma template. Try to fix the `height `of your **popup window**. You can see a difference compared to the Figma template. Please use the dimensions and follow the layout as defined in the Figma template to make them similar and give the best user experience :smile: 

- [ ] Please reduce the unnecessary spaces between your description and your technology list of your modal so that we have the same dimensions as in Figma template :+1: 

- [ ]  Please work on the **close button** to be the same as on the Figma template and more enjoyable. :smile: 


## Desktop version

### Figma template
![Screenshot from 2022-07-21 17-19-08](https://user-images.githubusercontent.com/6615394/180274853-d13d01b4-7710-4b5c-bf9a-f7ef7ac48a28.png)

### Your project
![Screenshot from 2022-07-21 17-18-56](https://user-images.githubusercontent.com/6615394/180274869-5ef4d901-8df3-4ef6-a773-4105e5fe3b9e.png)
![Screenshot from 2022-07-21 17-18-10](https://user-images.githubusercontent.com/6615394/180274873-f567d7d5-15ab-4557-b649-e2f0e6005f72.png)

## Mobile Version

### Figma mobile popup
![Screenshot from 2022-07-21 17-23-09](https://user-images.githubusercontent.com/6615394/180275557-7a4f76d8-76a2-4e98-b652-833b9cbbefb1.png)

### Your mobile popup
![Screenshot from 2022-07-21 17-23-00](https://user-images.githubusercontent.com/6615394/180275566-80541218-838d-41cb-a938-de2cbbfbd97c.png)